packages/docs-site/src/components/presenters/Masthead/index.js

```diff
@@ -26,6 +25,15 @@ const MastHead = ({ navItems }) => {
 
   return (
     <div className="masthead">
+      <Link
```

Comment:
Looks good for now - maybe in the future we could handle this with a flag or some config?

Reply:
Yeah, probably a smart idea. I guess after a period of time of `v2` being live, it might be an idea to use this banner to point to the legacy `v1` docs? Just a thought